scripts/ci/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Curious to know why this was needed

Reply:
Because I started to write a jest test but stopped, I think this can be removed because I didn't finish the test, and we really need to unblock the CI